### PR TITLE
Enhancements on some basic mif output

### PR DESCRIPTION
**Pull request by massfords**
## Checklist

- [ ] I have compiled and run my code on NICE or my hardware
- [ ] I've reviewed the **Files changed** tab
- [ ] Pull request is merged
- [ ] Mention all course staff in a comment

### CODE REVIEW

**Comment by massfords** [comment]
@stbenjam for your consideration

**Comment by DineshUCD** [comment]
Merging the new edits

File: asm/e93-assembler/README.md [hunk 1]
```diff
@@ -11,3 +11,9 @@ Running:
 - build as shown above
 - java -jar target/e93-assembler-1.0-SNAPSHOT.jar name-of-your-file.asm
 
+Running a single test:
+
+```
+./mvnw clean test -Dtest=AssemblerTest
```
Comment (stbenjam) [inline asm/e93-assembler/README.md]: We should turn on travis for the assembler, but we can do it as a separate PR later

Comment (stbenjam) [reply asm/e93-assembler/README.md]: https://github.com/CSCIE9X/csci-e-93-project/pull/3

**Comment by stbenjam** [comment]
Going to merge this so we have it for next semester, thanks for the enhancements @massfords

**Pull request merged**